examples/example_container_vulnerabilities_csv.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+from laceworksdk import LaceworkClient
+from datetime import datetime, timedelta
+import csv
+import json
```

Comment:
Nit. I'd like to follow PEP8 even for examples (https://www.python.org/dev/peps/pep-0008/#imports)
```suggestion
import csv
import json

from datetime import datetime, timedelta

from laceworksdk import LaceworkClient
```